lib/gui/app/modules/analytics.js

```diff
@@ -48,11 +48,17 @@ resinCorvus.install({
   }
 })
 
+let mixpanelSample = DEFAULT_PROBABILITY
+
 getConfig(configUrl)
   .then((config) => {
+    const mixpanel = _.get(config, [ 'analytics', 'mixpanel' ]) || {}
+    const probability = mixpanel.probability || DEFAULT_PROBABILITY
+    const validatedConfig = isClientEligible(probability) ? validateMixpanelConfig(mixpanel) : null
     resinCorvus.setConfigs({
```

Comment:
Do we need to call setConfigs if the config is null?

Reply:
I think so, let me check again on `resin-corvus`

Reply:
We need to do so in order to clear the queue of events that stacks, since we're loading Mixpanel deferred in order to retrieve the online config first

Reply:
This means that we need a `.catch` for this `getConfig()` or the mixpanel events will accumulate forever.

Reply:
Added